### DISCUSSION TITLE
Simple Web Interface using Nixie Stand-alone Controller

### DISCUSSION THREAD

**Discussion by bluong77** [tagyoureit/nodejs-poolController#998] [Q&A]
Hi,

During my pool construction, I made a stand-alone Nixie controller and follow the guides to do this as i always like to tinker with stuff.  I have only tested the connection to the equipment to see if communications between the SWG, Pumps, and Heater (Fireman Switch), and lights and everything seems to work.

After using my pool for a month, I realize i don't require much automation at all as my pool setup is simple and my pump is scheduled to ensure chlorine generation happens and the water has been turned over.  My intentions really are to be able to turn on the Heater / Change Pump Speeds while i am inside the house or somewhere on the road and for my wife to easily do this with a push of a button on an app.

I basically want my wife (using some sort of app) to just push one button to turn on/off the Heat/Pump when she wants to swim. 

So the question is, what is the easiest way (if there is a guide on this) where i can use some sort of automation app (that can be accessed from anywhere in the world, not just from your local home network?  Dash panel can only be accessed from your home network.  Do i need to create a VPN and also somehow integrate to Home Assistant.  Truthfully, i don't really know how to do either.. If this is the path, is there a step-by-step tutorial on doing this?  I can't possibily think everyone is using Dashpanel as their main interface to using the Pool day-in/day-out?

Not sure if this makes sense?

My alternative is to just sell my Nixie Stand-alone build and buy something like the Sonoff 4ch Wireless relay.  1ch for lights, 2nd and 3rd channel for 2 speeds of the pump, and 4th channel for Heater (Fireman switch).  With this, it would be a simple App that can be used outside of your home network.  Only thing with this is that i won't see info on the SWG or set a specific pump speed, or change the heater settings (probably never have to anyways).

Thoughts?

**Comment by rstrouse** [comment]
Home Assistant will give you this plus so much more including real time monitoring as well as secure access to nearly everything in your home.  You can also access dashPanel using something like ZeroTier.

**Reply by bluong77** [reply]
Great. I'll look into finding a tutorial on Home Assistant and integrating it with the Nixie Pool Controller.

**Reply by rstrouse** [reply]
There is a Home Assistant Community Store (HACS) integration that loads up all the entities you have defined.
